src/components/Banner/Banner.tsx

```diff
@@ -106,38 +106,29 @@ export const Banner = forwardRef<BannerHandles, BannerProps>(function Banner(
   }
 
   let actionMarkup;
-  if (action) {
-    if (newDesignLanguage) {
-      actionMarkup = (
-        <div className={styles.Actions}>
-          <ButtonGroup>
-            <div className={styles.PrimaryAction}>
-              {unstyledButtonFrom(action, {
-                className: styles.Button,
-              })}
-            </div>
+  if (action || secondaryAction) {
+    const primaryActionMarkup = action ? (
+      <div className={styles.PrimaryAction}>
+        {newDesignLanguage
+          ? unstyledButtonFrom(action, {
+              className: styles.Button,
+            })
+          : buttonFrom(action, {outline: true, size: buttonSizeValue})}
+      </div>
+    ) : null;
 
-            {secondaryAction && (
-              <SecondaryActionFrom action={secondaryAction} />
-            )}
-          </ButtonGroup>
-        </div>
-      );
-    } else {
-      actionMarkup = (
-        <div className={styles.Actions}>
-          <ButtonGroup>
-            <div className={styles.PrimaryAction}>
-              {buttonFrom(action, {outline: true, size: buttonSizeValue})}
-            </div>
+    const secondaryActionMarkup = secondaryAction ? (
+      <SecondaryActionFrom action={secondaryAction} />
+    ) : null;
 
-            {secondaryAction && (
-              <SecondaryActionFrom action={secondaryAction} />
-            )}
-          </ButtonGroup>
-        </div>
-      );
-    }
+    actionMarkup = (
```

Comment:
We don't really need this to be a let anymore, we can just do:
```
const actionMarkup = action || secondaryAction ? (<div className={styles.Actions}>...etc) : null;
```

Then we can also get rid of the if statement above. Might be slightly cleaner 🤷‍♂️